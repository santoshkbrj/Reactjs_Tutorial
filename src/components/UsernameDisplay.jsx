import React from 'react'

function UsernameDisplay({ username }) {
    return (
        <div>{username}</div>
    )
}

export default UsernameDisplay