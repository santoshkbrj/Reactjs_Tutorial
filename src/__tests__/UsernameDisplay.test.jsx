import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import UsernameDisplay from "../components/UsernameDisplay";

describe('UsernameDisplay', () => {
    it('should render username', () => {
        const result = render(<UsernameDisplay />);
        expect(result.container).toMatchSnapshot();
    })
})