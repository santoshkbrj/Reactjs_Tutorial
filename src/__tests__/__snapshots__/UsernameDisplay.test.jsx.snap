// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UsernameDisplay > should render username 1`] = `
<div>
  <div />
</div>
`;
